crates/core_arch/src/x86/test.rs

```diff
@@ -139,3 +139,47 @@ pub unsafe fn assert_eq_m512d(a: __m512d, b: __m512d) {
         panic!("{:?} != {:?}", a, b);
     }
 }
+
+#[cfg(all(target_arch = "x86_64", windows))]
+#[target_feature(enable = "amx-tile")]
+pub(crate) fn _init_amx() {}
+
+#[cfg(all(target_arch = "x86_64", not(windows)))]
```

Comment:
This should specifically be `cfg(target_os = "linux")`. The syscalls are specific to Linux and won't work on other OSes like BSD.